pkgs/applications/misc/mop/default.nix

```diff
@@ -10,6 +10,10 @@ buildGoPackage rec {
 
   preConfigure = ''
     for i in $(find . -type f);do
```

Comment:
rather than using a subshell and a find, how about a glob, it would be more precise
```
for i in **/*.go ;do
...
done
```
you wouldn't need the case expression

Reply:
Sounds good. Did `*.go **/*.go` to also capture `$PWD`.